src/Microsoft.Azure.WebJobs.Host/Hosting/WebJobsHostBuilderExtensions.cs

```diff
@@ -135,6 +140,38 @@ public static IHostBuilder ConfigureWebJobs(this IHostBuilder builder, Action<Ho
             return builder;
         }
 
+        /// <summary>
+        /// Configures the specified <see cref="IHostBuilder"/> as a scale manager host.
+        /// </summary>
+        /// <param name="builder">The <see cref="IHostBuilder"/> to configure.</param>
+        /// <param name="configure">Configuration action to perform as part of service configuration.</param>
+        /// <param name="configureScaleOptions">Configuration action for <see cref="ScaleOptions"/>.</param>
+        /// <returns>The <see cref="IHostBuilder"/>.</returns>
+        public static IHostBuilder ConfigureWebJobsScale(this IHostBuilder builder,
+            Action<HostBuilderContext, IWebJobsBuilder> configure,
+            Action<ScaleOptions> configureScaleOptions)
+        {
+            builder.ConfigureServices((context, services) =>
+            {
+                WebJobsBuilder webJobsBuilder = new WebJobsBuilder(services);
+                configure?.Invoke(context, webJobsBuilder);
+
+                if (configureScaleOptions != null)
+                {
+                    services.Configure(configureScaleOptions);
+                }
+
+                services.AddFeatureManagement();
+                services.TryAddSingleton<IScaleMonitorManager, ScaleMonitorManager>();
+                services.TryAddSingleton<ITargetScalerManager, TargetScalerManager>();
+                services.TryAddSingleton<IScaleManager, ScaleManager>();
+                services.TryAddSingleton<IHostedService, ScaleMonitorService>();
```

Comment:
We need to add `ScaleMonitorService` in function host. As implementation is internal we need to also call `ConfigureWebJobsScale` or add a new method.